src/NewTools-Debugger/StDebuggerContextInteractionModel.class.st

```diff
@@ -24,9 +32,19 @@ StDebuggerContextInteractionModel >> bindingOf: aString [
 
 	"we can not call #lookupVar: without checking first as it would create the variable"
 
-	^ (self hasBindingOf: aString)
-		  ifTrue: [ (context lookupVar: aString) asDoItVariableFrom: context ]
-		  ifFalse: [ nil ]
+	^ (self hasBindingInContextOf: aString)
+		  ifTrue: [ 
+		  (context lookupVar: aString) asDoItVariableFrom: context ]
+		  ifFalse: [ 
+			  (self hasBindingInInteractionModelOf: aString)
+				  ifTrue: [ self bindings associationAt: aString ]
+				  ifFalse: [ nil ] ]
```

Comment:
Could this be simplified by something like
```Smalltalk
self bindings associationAt: aString ifAbsent:[nil]
```
?

Reply:
Yes, you're right.

You can replace

```Smalltalk
(self hasBindingInInteractionModelOf: aString)
 				  ifTrue: [ self bindings associationAt: aString ]
 				  ifFalse: [ nil ]
```

by 

```Smalltalk
self bindings associationAt: aString ifAbsent:[nil]
```

Reply:
done